build.cake

```diff
@@ -584,7 +584,7 @@ Task("SRGen")
                 continue;
             }
 
-            var srgenPath = System.IO.Path.Combine(toolsFolder, "Microsoft.Data.Tools.StringResourceTool", "lib", "netcoreapp2.2", "srgen.dll");
+            var srgenPath = System.IO.Path.Combine(toolsFolder, "Microsoft.Data.Tools.StringResourceTool", "lib", "netcoreapp3.1", "srgen.dll");
```

Comment:
FYI this isn't going to work since the 2.0.0 version of the tool wasn't compiled for netcore 3.1. I have a PR out fixing this already with an updated version of the tool.

Reply:
I'll wait for your build to pass and be merged in main. After that, I will pull and add the changes to my branch as well.